### PR TITLE
[WIP] Support sepolia testnet

### PR DESCRIPTION
**Pull request by TTNguyenDev**
https://app.dework.xyz/nation3/development-guild-43499?taskId=28308d89-f718-467d-9306-3efc01daf7ed

### CODE REVIEW

File: deployments/sepolia.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+ "nationToken": "0x23Ca3002706b71a440860E3cf8ff64679A00C9d7",
+ "veNationToken": "0x8100e77899C24b0F7B516153F84868f850C034BF",
+ "balancerLPToken": "0x6417755C00d5c17DeC196Df00a6F151E448B1471",
+ "lpRewardsContract": "0x534AB4F195Ac166B78d9edCdbeA04802392711aA",
+ "nationDropContracts": [
+  "0xfa5742931300D9CcFB8FE789dbd72326DFf770fB",
+  "0x32ebd2b43fdF9Ac035Cb9947E5086F0Ca53be573"
+ ],
```
Comment (aahna-ashina) [inline deployments/sepolia.json]: We will not be doing more airdrops, so can we exclude these contracts?
```suggestion
```

File: .env.sepolia.sample [hunk 2]
```diff
@@ -0,0 +1,7 @@
+RPC_URL="https://rpc.sepolia.org"
+DEPLOYER_PRIVATE_KEY=""
+
+NATION_SUPPLY="42069000000000000000000"
+
+AIRDROP_ROOT="0xed145aa219b18aa3f2dc56afb2c4e0b148e429ca93b9c5f2c7a29d2101685aee"
+AIRDROP_AMOUNT="314000000000000000000"
```
Comment (aahna-ashina) [inline .env.sepolia.sample]:
```suggestion
```

File: deployments/sepolia.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+ "nationToken": "0x23Ca3002706b71a440860E3cf8ff64679A00C9d7",
```
Comment (aahna-ashina) [inline deployments/sepolia.json:2]: @TTNguyenDev Please also _verify_ the contracts:  https://sepolia.etherscan.io/address/0x23Ca3002706b71a440860E3cf8ff64679A00C9d7#code

> verify

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
Contract verification missing

File: deployments/sepolia.json (hunk 3 above)
Comment (TTNguyenDev) [reply deployments/sepolia.json:2]: Okay sure

**Comment by TTNguyenDev** [comment]
@aahna-ashina Resolved

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**